hack/verify-boilerplate.sh

```diff
@@ -39,12 +43,12 @@ files=($(find . -type f -not \( \
         -o -path './vendor/*' \
         -o -path './pkg/client/*' \
         -o -path './*/.terraform/*' \
+        -o -path './tests/images/*/*' \
```

Comment:
`./tests/images/*/*` matches the docker image build directories which we should ignore